clientd3d/game.h

```diff
@@ -115,4 +115,6 @@ M59EXPORT player_info *GetPlayerInfo(void);
 M59EXPORT void SetActiveStatGroup(int stat_group);
 M59EXPORT int GetActiveStatGroup(void);
 
+M59EXPORT const room_type& getCurrentRoom();
```

Comment:
Not sure this should be M59EXPORT.  This means you are exporting it to modules and using it there, but I don't see any usage.